metadb/scrapers/musicbrainz.py

```diff
@@ -0,0 +1,33 @@
+import getpass as gt
+import sys
+import musicbrainzngs as mb
+
+username = raw_input('Enter MusicBrainz Username: ')
+password = gt.getpass('Password for {}: '.format(username))
+mb.auth(username,password)
+
+mb.set_useragent(
+    "python-metadb-musicbrainz-scrape",
+    "0.1",
+    "https://github.com/MTG/metadb")
+
+
+
+def scrape(mbid, artist, title):
+    try:
+        resultID = mb.get_recording_by_id(mbid,
+                                          includes=["artists", "releases", "annotation", "tags"],
+                                          release_status=["official"],
+                                          release_type=["album", "single", "ep", "broadcast", "other", "compilation"])
+        
+        resultArtist = mb.search_artists(artist=artist,
```

Comment:
The artist and title only need to be used if the service that we are scraping does not support lookup by Musicbrainz ID. As we are accessing musicbrainz, we can do that :). So I don't think we need `resultArtist` or `resultTitle`